doc/exts/pylint_messages.py

```diff
@@ -318,6 +318,13 @@ def _generate_single_message_body(message: MessageData) -> str:
 .. caution::
   This message is disabled by default. To enable it, add ``{message.name}`` to the ``enable`` option.
 
+"""
+    if message.id.startswith("I"):
+        body += f"""
+.. caution::
+  By default, this message will not fail the execution (pylint will return 0).
+  To make pylint fail use the ``--fail-on={message.id}`` option.
```

Comment:
Should we mention (either instead or in addition to) `--fail-on=I`?

Reply:
Done and rebased because there's an unrelated commit I don't want to squash.